docs/desktop-frontend-testing.md

```diff
@@ -63,3 +34,12 @@ await enrollmentPromise;
 // Finishing up
 await doExperimentCleanup();
 ```
+
+## Testing with a live Nimbus recipe
+
+If you already published an experiment through Nimbus then you will want to test enrollment in the browser.
+
+You need to copy paste the following address and fill in the required details.
+These steps assume you used the Preview functionality in Experimenter and the recipe is not yet deployed to production.
```

Comment:
Maybe can you do something like:

You need to copy paste the following URL and fill in the required details.

For experiments that are already live:

`about:studies?optin_slug=<YOUR_EXPERIMENT_SLUG>&optin_branch=<BRANCH_SLUG_TO_ENROLL>`

For experiments that are in "preview" mode:
`about:studies?optin_slug=<YOUR_EXPERIMENT_SLUG>&optin_branch=<BRANCH_SLUG_TO_ENROLL>&optin_collection=nimbus-preview`